source/v4/channels.yaml

```diff
@@ -80,7 +78,97 @@
           $ref: '#/responses/Unauthorized'
         '403':
           $ref: '#/responses/Forbidden'
-
+  '/channels/{channel_id}':
+    get:
+      tags:
+        - channels
+      summary: Get a channel
+      description: |
+        Get channel from the provided channel id string.
+        ##### Permissions
+        `read_channel` permission for the channel.
+        `manage_system` permission for the channel.
```

Comment:
You can actually remove the `manage_system` permission since anyone who has that permission will already have `read_channel` for every channel. My mistake if I included `manage_system` in the spreadsheet